e2e/libs/workload/pod.py

```diff
@@ -134,7 +134,8 @@ def delete_pod(name, namespace='default', wait=True):
         try:
             core_api.delete_namespaced_pod(name=name, namespace=namespace, grace_period_seconds=0)
             if wait:
-                wait_delete_pod(name)
+                wait_delete_pod(name, namespace)
+            break
```

Comment:
Do we need the `break` here? If the pod has been deleted, `delete_namespaced_pod` function will throw `404 Not found` exception, and be caught by:
```
except rest.ApiException as e:
    if e.status == 404:
        logging(f"Deleted pod {name} in namespace {namespace}")
        return
``` 
then this `delete_pod` function can successfully return.

Reply:
It's because longhorn-manager pod will be automatically recreated immediately.